docroot/sites/classrooms.uiowa.edu/modules/classrooms_core/classrooms_core.module

```diff
@@ -192,6 +227,15 @@ function classrooms_core_preprocess_field(&$variables, $hook) {
       }
       $variables['attributes']['class'] .= 'fa-li';
       break;
+
+    case 'field_room_guide':
+      $link_items = $variables['items'];
+      foreach ($link_items as &$item) {
+        $item['content']['#options']['attributes']['class'][] = 'bttn bttn--light-font bttn--transparent bttn--full';
+        $item['content']['#title'] = Markup::create($item['content']['#title'] . '<i role="presentation" class="fas fa-arrow-right"></i>');
```

Comment:
See above comment about field template.

Reply:
This has been removed and moved to the template.